akka-docs/src/main/paradox/stream/operators/Compression/deflate.md

```diff
@@ -0,0 +1,30 @@
+# Compression.deflate
+
+Creates a flow that deflate-compresses a stream of ByteString.
+
+@ref[Compression operators](../index.md#compression-operators)
+
+@@@div { .group-scala }
+
+## Dependency
+
+This operator is included in:
+
+@@dependency[sbt,Maven,Gradle] {
+  group="com.typesafe.akka"
+  artifact="akka-stream_$scala.binary_version$"
+  version="$akka.version$"
+}
```

Comment:
We do for `ActorFlow/ask.md` and `ActorSink/actorRef.md`. There it makes sense since they rely on `akka-stream-typed`. I agree they're not needed for operators included in akka-stream itself, so I agree they can be removed from this PR (and from `Source/range.md`).